README.md

```diff
@@ -3,21 +3,19 @@
 [![crates.io](https://img.shields.io/crates/v/rustc-hash.svg)](https://crates.io/crates/rustc-hash)
 [![Documentation](https://docs.rs/rustc-hash/badge.svg)](https://docs.rs/rustc-hash)
 
-A speedy hash algorithm used within rustc. The hashmap in liballoc by
-default uses SipHash which isn't quite as speedy as we want. In the
-compiler we're not really worried about DOS attempts, so we use a fast
-non-cryptographic hash.
-
-This is the same as the algorithm used by Firefox -- which is a
-homespun one not based on any widely-known algorithm -- though
-modified to produce 64-bit hash values instead of 32-bit hash
-values. It consistently out-performs an FNV-based hash within rustc
-itself -- the collision rate is similar or slightly worse than FNV,
-but the speed of the hash function itself is much higher because it
-works on up to 8 bytes at a time.
+A speedy, non-cryptographic hashing algorithm used by `rustc` and Firefox.
+The [hash map in `std`](https://doc.rust-lang.org/std/collections/struct.HashMap.html) uses SipHash by default, which provides resistance against DOS attacks.
+These attacks aren't as much of a concern in the compiler so we prefer to use the quicker, non-cryptographic Fx algorithm.
+
+The Fx algorithm is a unique one used by Firefox. It is fast because it can hash eight bytes at a time.
+Within `rustc`, it consistently outperforms FNV-based algorithms.
+The collision rate is similar or slightly worse than FNV.
```

Comment:
```suggestion
Within `rustc`, it consistently outperforms every other tested algorithm (for example, FNV).
The collision rate is similar or slightly worse than others.
```
There have been a lot of benchmarks and fxhash has always come out on the winning side.

Reply:
Changed. "(such as FNV)" sounds more suitable to me.